content/admin/configuration/configuring-the-ip-address-using-the-virtual-machine-console.md

```diff
@@ -14,3 +14,10 @@ versions:
 4. Configure options for the protocol you chose.
   ![Menu with IP protocol options](/assets/images/enterprise/network-configuration/network-settings-selection.png)
 {% data reusables.enterprise_installation.vm-console-done %}
+
+---
```

Comment:
@casals Could you update this to use the callouts described in the [content style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#formatting-callouts) and move it to the top of the procedural? That way users will have all of the information before they start working through the steps. Thank you!

Reply:
Absolutely. Fair warning - I inserted quite a few notes in yesterday's PRs, so when you come across them again could you please just tag me + "callout" or something similar? I'll adjust all of them. Thanks!

Reply:
Also - whenever I used notes at the beginning of their related context I labeled them as "Before you start", IIRC - could you please tag me on these as well? Consistency > ALL. :) Thanks!